deployment/morph/overlays.nix

```diff
@@ -0,0 +1,16 @@
+{
+  # journalbeat 5.6.0 has a bug so we need to upgrade but its not in stable nixpkgs yet
```

Comment:
Could you turn this comment into more of an action item? 
```
TODO: drop once <some link> is in the pinned nixpkgs
```

Reply:
yeah I think we don't need this any more, removing